templates/express.txt

```diff
@@ -0,0 +1,14 @@
+import express from 'express'
+import dotenv from 'dotenv'
+
+dotenv.config()
+
+const app = express()
+
+const PORT = process.env.PORT || 8000
+
+app.use('/', (_req, res) => {
+  res.send({ hello: 'world' })
```

Comment:
I think the more idiomatic way would be:

```suggestion
  res.json({ hello: 'world' })
```

This will ensure the default status of 200 and set the content type header appropriately.